app/src/androidTest/java/com/droidhats/campuscompass/views/MapFragmentTest.kt

```diff
@@ -75,4 +83,35 @@ class MapFragmentTest {
         //Ensuring the text of the toggle just clicked is indeed LOY
         onView(withId(R.id.toggleButton)).check(matches(withText("LOY")))
     }
+
+    @Test
+    fun test_additionalMenuBar(){
+        GlobalScope.launch {
+            delay(2000) //to allow bottomsheet to load
```

Comment:
Can we change these lines to *Thread.sleep(1000)* instead?
I noticed that it causes 3 new imports in comparison to using Thread.sleep.
Also when I initially ran the tests, the test_SwitchToggle() wasn't able to finish because of a process crash. After changing the lines, all tests were passing.

Reply:
Yes ! Indeed, this solution does solve the crash! Thank you